ui/lib/apps/SlowQuery/pages/Detail/DetailTabTxn.tsx

```diff
@@ -24,7 +24,7 @@ export default function TabCopr({ data }: ITabTxnProps) {
       value: getValueFormat('bytes')(data.write_size || 0, 1),
     },
     {
-      key: 'prewrite_regions',
+      key: 'prewrite_region',
```

Comment:
In slow query table, this column names 'Prewrite_region', not 'Prewrite_regions'.